website/docs/FAQ.md

```diff
@@ -108,3 +107,25 @@ prompt += termination_notice
 ```
 
 **Note**: This workaround gets the job done around 90% of the time, but there are occurences where the LLM still forgets to terminate the conversation.
+
+## ChromaDB fails in codespaces because of old version of sqlite3
+
+(from [issue #251](https://github.com/microsoft/autogen/issues/251))
+
+Code examples that use chromadb (like retrieval) fail in codespaces due to a sqlite3 requirement.
+```
+>>> import chromadb
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+  File "/home/vscode/.local/lib/python3.10/site-packages/chromadb/__init__.py", line 69, in <module>
+    raise RuntimeError(
+RuntimeError: Your system has an unsupported version of sqlite3. Chroma requires sqlite3 >= 3.35.0.
+Please visit https://docs.trychroma.com/troubleshooting#sqlite to learn how to upgrade.
+```
+
+Workaround:
+1. `pip install pysqlite3-binary`
+2. `mkdir ../../home/vscode/.local/lib/python3.10/site-packages/google/colab`
```

Comment:
```suggestion
2. `mkdir /home/vscode/.local/lib/python3.10/site-packages/google/colab`
```

Reply:
After rebuilding my codespace (I had probably broken it somehow), this worked fine for me. Thank you